tests/bluetooth/bsim/audio/src/tbs_client_test.c

```diff
@@ -180,6 +426,79 @@ static void test_ccid(void)
 	}
 }
 
+static void test_signal_strength(uint8_t index)
+{
+	int err;
+
+	UNSET_FLAG(signal_strength);
+
+	printk("%s\n", __func__);
+
+	err = bt_tbs_client_read_signal_strength(default_conn, index);
+	if (err != 0) {
+		FAIL("Read signal strength failed (%d)\n", err);
+		return;
+	}
+
+	WAIT_FOR_FLAG(signal_strength);
+
+	printk("Client read signal strength test success\n");
+}
+static void test_technology(uint8_t index)
```

Comment:
```suggestion
}

static void test_technology(uint8_t index)
```

Applies multiple places below

Reply:
Fixed :)